test/functional/test_framework/test_node.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, timeout_factor, bitc
         self.args = [
             self.binary,
             "--datadir={}".format(datadir_path),
-            "run",
+            "regtest",
```

Comment:
Cool I kind of like the command-based interface.

Reply:
Glad to hear! I was afraid that no one would like this approach.